js/bytex.js

```diff
@@ -909,76 +1020,61 @@ module.exports = class bytex extends Exchange {
         //     }
         //
         const marketId = this.safeString (order, 'symbol');
+        const symbol = this.safeSymbol (marketId, market, '-');
+        const id = this.safeString (order, 'id');
         const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
-        const meta = this.safeValue (order, 'meta');
-        const filled = this.safeString (order, 'filled');
+        const type = this.safeString (order, 'type');
+        const side = this.safeString (order, 'side');
+        const price = this.safeString (order, 'price');
         const amount = this.safeString (order, 'size');
+        const filled = this.safeString (order, 'filled');
+        const status = this.parseOrderStatus (this.safeString (order, 'status'));
         return this.safeOrder ({
-            'id': this.safeString (order, 'id'),
+            'id': id,
             'clientOrderId': undefined,
             'timestamp': timestamp,
             'datetime': this.iso8601 (timestamp),
             'lastTradeTimestamp': undefined,
-            'status': this.parseOrderStatus (this.safeString (order, 'status')),
-            'symbol': this.safeSymbol (marketId, market, '-'),
-            'type': this.safeString (order, 'type'),
+            'status': status,
+            'symbol': symbol,
+            'type': type,
             'timeInForce': undefined,
-            'postOnly': this.safeValue (meta, 'post_only'),
```

Comment:
Why was this removed? It correctly added `true` if this was a post only order when I was testing it